src/Xamarin.Android.Build.Tasks/Tasks/GetAdditionalResourcesFromAssemblies.cs

```diff
@@ -319,13 +323,13 @@ public override bool Execute ()
 					foreach (var ca in resolver.GetAssembly (assemblyItem.ItemSpec).CustomAttributes) {
 						switch (ca.AttributeType.FullName) {
 						case "Android.IncludeAndroidResourcesFromAttribute":
-							AddAttributeValue (androidResources, ca, "XA5206", "{0}. Android resource directory {1} doesn't exist.", true, fullPath);
+							AddAttributeValue (androidResources, ca, "XA5206", "{0}. (Details: Android resource directory {1} doesn't exist.)", true, fullPath);
```

Comment:
Yeah, I debated with myself on the best option.  I think it could also be OK to log an additional separate XA5207 that lists the missing file name, but I wasn't sure about the conventions for when or when not to log 2 separate messages for one error code.

One thing I forgot to check was whether there was any precedent for using something similar to these parentheses and "Details:" text in other error messages.  If not, then another approach might indeed be better.  The issue I was aiming to address is that the error messages are a bit more confusing when there isn't a clear separation between the first part of the message and the details about the missing file.

For example, compare:

> error XA5207: Unable to restore Java resources for package 'Foo'. The authors of the package should set the SourceUrl property or InstallInstructions property on the Java.Interop.JavaLibraryReferenceAttribute. (Details: Java library file foo.jar doesn't exist.)

To:

> error XA5207: Unable to restore Java resources for package 'Foo'. The authors of the package should set the SourceUrl property or InstallInstructions property on the Java.Interop.JavaLibraryReferenceAttribute. Java library file foo.jar doesn't exist.

Or compare:

> error XA5207: Installation instructions: Please download foo.jar from http://example.com/foo.jar and copy it into the Android app project directory. (Details: Java library file foo.jar doesn't exist.)

to:

> error XA5207: Installation instructions: Please download foo.jar from http://example.com/foo.jar and copy it into the Android app project directory. Java library file foo.jar doesn't exist.